### ISSUE TITLE
[css-cascade] Apparent contradiсtion between definitions of the cascade and the output of the cascade

### ISSUE THREAD

**Issue by SelenIT** [w3c/csswg-drafts#6996]
The 1st paragraph of the _§6. Cascading_ in both Level 4 and Level 5 states (emphasis mine):

> The _cascade_ takes an unordered list of declared values for a given property on a given element, sorts them by their declaration’s precedence as determined below, and _outputs_ **a single cascaded value.**

At the same time, the last paragraph of _§6.1. Cascade Sorting Order_ says that
> The _output of the cascade_ is **a** (potentially empty) **sorted list of declared values** for each property on each element.

These statements seem to contradict each other, which is confusing. Are "the output of the cascade" and the thing that "the cascade outputs" not the same thing? Shouldn't this be clarified for consistency?

**Comment by tabatkins** [comment]
Yes, "cascaded value" and "output of the cascade" are not the same thing. If you follow the "cascaded value" link to its definition, you see:

> The cascaded value represents the result of the cascade: it is the declared value that wins the cascade (is sorted first in the output of the cascade). If the output of the cascade is an empty list, there is no cascaded value.

The "output of the cascade" is a list; the "cascaded value" is one of the items in that list, or nothing if it's empty.

That said, the wording there is indeed somewhat confusing. Ideas, @fantasai?

**Comment by tabatkins** [comment]
Wondering if just "cascade list" might be an appropriate replacement for "output of the cascade"

**Comment by SelenIT** [comment]
To me, it seems that the confusion comes from using the term "the cascade" in a broad sense (the whole algorithm of making the single value from multiple inputs) and in a narrower sense (the algorithm of sorting the initially unordered list of values) in the same text. Since the section that introduces the latter definition is titled "Cascade _Sorting_ Order", maybe just adding the word "sorting" ("the output of the cascade _sorting_") would clear the confusion?
